src/main/kotlin/racingCar/controller/RacingController.kt

```diff
@@ -0,0 +1,49 @@
+package racingCar.controller
+
+import racingCar.model.Car
+import racingCar.model.RaceResult
+import java.util.Random
+
+/**
+ * 레이싱을 시작하고, 계산하는 클래스
+ * Created by Jaesungchi on 2022.05.07..
+ */
+class RacingController {
+    private val carList: MutableList<Car> = mutableListOf()
+    val raceResults: MutableList<RaceResult> = mutableListOf()
+
+    fun startGame(numberOfCars: Int, numberOfRaces: Int) {
+        repeat(numberOfCars) {
+            carList.addAll(listOf(Car()))
```

Comment:
`Car`를 list로 감싸지 않아도 괜찮을 것 같아요~

```suggestion
            carList.add(Car())
```

Reply:
아이고 이런 실수를... 감사합니다 ㅠㅠ